tools/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/lightningnetwork/lnd/tools
 
-go 1.22.6
+go 1.23.6
```

Comment:
Seems like the linter comes with new rules in 1.23.6, see failures. Also, there's another instance of `1.22.11` in `lnrpc/gen_protos_docker.sh`.